lightning/src/ln/channel.rs

```diff
@@ -1093,6 +1094,14 @@ impl<Signer: Sign> Channel<Signer> {
 		// of 1.1 sat/vbyte and a receiver that wants 1.1 rounded up to 2. Thus, we always add 250
 		// sat/kw before the comparison here.
 		if feerate_per_kw + 250 < lower_limit {
+			if let Some(cur_feerate) = cur_feerate_per_kw {
+				if feerate_per_kw > cur_feerate {
+					log_error!(logger,
+						"Accepting low feerate that may prevent us from closing this channel because its higher than what we have now. Had {} s/kW, now {} s/kW.",
```

Comment:
nit:
```suggestion
						"Accepting low feerate that may prevent us from closing this channel because it's higher than what we have now. Had {} s/kW, now {} s/kW.",
```